messages.proto

```diff
@@ -285,6 +285,25 @@ message StepPositionsResponse {
     string error = 2;
 }
 
+/// Request for getting Implementation file list
+message ImplementationFileListRequest {
+}
+
+/// Response for getting Implementation file list
+message ImplementationFileListResponse {
+    /// Get the implementation directory list for folder path
+    repeated string implementationFilePaths = 1;
+}
+
+/// Request for injecting code snippet into implementation file
+message StubImplementationCodeRequest {
+    /// Path of the file where the new stub implementation will be added
+    string implementationFilePath = 1;
+
+    /// List of step texts for which we want to generate stub implementation
+    repeated StepValidateRequest steps = 2;
```

Comment:
Why are we using stepValidate request here?

Reply:
We are using this since it has all the fields required, so I thought of reusing this instead of creating a new request. Since we are sending step data from gauge but the response it receives is different so I used a different response for that